contracts/Octobay.sol

```diff
@@ -391,44 +320,22 @@ contract Octobay is Ownable, ChainlinkClient, BaseRelayRecipient {
             // Do other permission checks here, e.g. oracle calls
         }
         require(hasPermission, "You don't have permission to set governance tokens for issues");
-        govTokenByIssueId[_issueId] = OctobayGovToken(_govTokenAddress);
+        issueDepositStorage.setGovTokenForIssue(_issueId, _govTokenAddress);
         emit SetGovTokenForIssueEvent(msg.sender, _issueId, _govTokenAddress, _projectId);
     }
 
     function depositEthForIssue(string calldata _issueId) public payable {
-        require(msg.value > 0, 'You must send ETH.');
-        require(issueStatusByIssueId[_issueId] == IssueStatus.OPEN, 'Issue is not OPEN.');
-
-        nextIssueDepositId++;
-        issueDeposits[nextIssueDepositId] = IssueDeposit(
-            msg.sender,
-            msg.value,
-            _issueId
-        );
-        issueDepositIdsByIssueId[_issueId].push(nextIssueDepositId);
-        issueDepositIdsBySender[msg.sender].push(nextIssueDepositId);
-        issueDepositsAmountByIssueId[_issueId] += msg.value;
-        emit IssueDepositEvent(msg.sender, msg.value, _issueId, nextIssueDepositId);
+        // Forwards eth payment to issueDepositStorage to hold
+        uint256 issueDepositId = issueDepositStorage.depositEthForIssue{value: msg.value}(_issueId, msg.sender);
+        emit IssueDepositEvent(msg.sender, msg.value, _issueId, issueDepositId);
```

Comment:
In the other places we emit events in the respective `OctobayStorage` contract. Should be the same here I'd say.